packages/fe-fpm-writer/package.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "@sap-ux/fe-fpm-writer",
     "description": "SAP Fiori elements flexible programming model writer",
-    "version": "0.35.15",
+    "version": "0.35.14",
```

Comment:
`0.35.15` is already on npmjs https://www.npmjs.com/package/@sap-ux/fe-fpm-writer